tests/inference/qnn_test.py

```diff
@@ -475,41 +496,125 @@ def generate_resolvers():
       } for r in bitstring_resolvers]
 
     # hamiltonian energy
-    num_layers = 5
+    num_layers = 1
+    random.seed(self.python_random_seed)
     bits = random.sample(range(1000), self.num_bits)
-    units = random.sample(range(1, 100), num_layers)
-    activations = random.sample([
-        "elu", "exponential", "gelu", "hard_sigmoid", "linear", "relu", "selu",
-        "sigmoid", "softmax", "softplus", "softsign", "swish", "tanh"
-    ], num_layers)
+    units = [2] * num_layers
+    activations = ["linear"] * num_layers
```

Comment:
why only linear now?

Reply:
Restricted during debugging and forgot to add things back.  Restored to all activations now.